### DISCUSSION TITLE
[Feature Proposal] Add `.contents` for `display: contents;`

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [tailwindlabs/tailwindcss#1691] [Ideas]
This is missing from https://tailwindcss.com/docs/display/ and I found myself adding it to multiple projects:

```
.contents {
  display: contents;
}
```

**Comment by magicspon** [comment]
+1

**Comment by jenstornell** [comment]
Because other display properties are set as classes like `block` and `grid`, it should probably be just `contents`.

```html
<div class="contents">
  This row will be ignored
</div>
```

**Comment by LeoniePhiline** [comment]
@jenstornell As already proposed.
